packages/typescriptlang-org/src/components/layout/Sidebar.scss

```diff
@@ -143,7 +143,7 @@ nav#sidebar {
       }
 
       &.highlight a {
-        color: $ts-main-blue-color;
+        color: #2866a9;
```

Comment:
Where does this color come from?

Reply:
To ensure that the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds, this is the suggested color by Accessibility Insights keeping the background color, font size and font weight all the same.

Reply:
This doesn't feel right... Before:

![image](https://github.com/microsoft/TypeScript-Website/assets/5341706/bb601529-271e-4424-8437-3a00ad6d7954)

After:

![image](https://github.com/microsoft/TypeScript-Website/assets/5341706/e81b9013-cc77-41b5-a636-5e85504544f1)

This seems like we're getting less contrast, not more...

Reply:
Might be dark mode vs light mode too - it's been a few years but I think those are changed via those constants

Reply:
Yeah that's true, it has to do with the dark vs light mode. Let me see what to do about that.

Reply:
Yeah, I think that if we're tweaking colors, we probably need to be introducing new variables and using them rather than hardcoding color values in specific places. e.g. we could include a "ts-sidebar-link" color or something for this, right?

Reply:
I'm confused, we added a variable, but did it actually get applied only to the light theme?